OSDOCS-17165-replace-unhealthy-etc CQA (#117026)
openshift/openshift-docs: modules/restore-replace-stopped-etcd-member.adoc

.Prerequisites

* You identified the unhealthy etcd member.
* You verified that either the machine is not running or the node is not ready.
+
[IMPORTANT]
====

> You must wait if you power off other control plane nodes. The control plane nodes must remain powered off until the replacement of an unhealthy etcd member is complete.

Reviewer: ```suggestion
* Do not power on other control plane nodes until the unhealthy etcd member replacement is complete.
```

Reviewer: I would just add this as a prerequisite

====
+
* You confirmed access to the cluster as a user with the `cluster-admin` role.
* You created an etcd backup before replacing the unhealthy etcd member.
+
[IMPORTANT]
====

> Without a recent etcd backup, you might not be able to restore the cluster if replacement fails.

Reviewer: I can't say that I've ever seen a callout inside a prerequisites and immediately following one that tells you to take the backup. If it were me I'd cut this IMPORTANT entirely.

Author: Note no longer there

+------------------+---------+------------------------------+---------------------------+---------------------------+
----
+

> Take note of the ID and the name of the unhealthy etcd member because you need these values later in the procedure. The `etcdctl endpoint health` command continues to list the removed member until replacement is complete and a new member is added.

Reviewer: It took me a little to follow where he `etcdctl endpoint health` command came from and this perhaps is a good argument for why I'd leave the docs nested the way that they are.

Author: Can we remove the second sentence with the `etcdctl endpoint health` command. It isn't used anywhere in this module. It's used in the replace-stopped-baremetal-etcd-member and restore-replace-crashlooping-etcd-member modules.  Just a thought

clustername-8qw5l-worker-us-east-1c-pkg26   Running   m4.large    us-east-1   us-east-1c   3h28m   ip-10-0-170-181.ec2.internal   aws:///us-east-1c/i-06861c00007751b0a   running
----
+
In the example output, `clustername-8qw5l-master-0` is the control plane machine for the unhealthy node `ip-10-0-131-183.ec2.internal`. Its `STATE` is `stopped`.

.. Delete the machine of the unhealthy member by running the following command:
+
[source,terminal]
----
$ oc delete machine -n openshift-machine-api clustername-8qw5l-master-0
----
+

> In the command, replace `clustername-8qw5l-master-0` with the control plane machine name for the unhealthy node that you identified in the example output above.

Reviewer: I'd put this back. You can't have directional language "above".

    > new-master-machine.yaml
----
+

> In the command, replace `clustername-8qw5l-master-0` with the control plane machine name for the unhealthy node that you identified in the example output above.

Reviewer: Same thing here.
